code/search/SearchQuery.php

```diff
@@ -21,19 +21,33 @@ class SearchQuery extends ViewableData
     public $require = array();
     public $exclude = array();
 
+    /**
+     * @var SearchCriteriaInterface[]
+     */
+    public $criteria = array();
```

Comment:
I think using public properties here is something we'd like to move away from. Can we use getters and setters instead and make this protected?